.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
+

```

Comment:
I think you need to rebase to master - these files are already in. If you could do a round on this PR and ensure that only the intended files are changed that would be good, cause there are a bunch of seemingly unrelated changes to this PR.

Reply:
rebased with 68b3ee8